src/framework/theme/components/toggle/toggle.component.scss

```diff
@@ -0,0 +1,34 @@
+@import '../../styles/core/mixins';
+
+:host {
+  display: inline-flex;
+  outline: none;
+
+  .toggle-label {
```

Comment:
Please move styles outside of `host` to reduce selector weight. Also `.toggle-lable > .toggle ...` nesting is unnecessary here and could be flattened since styles encapsulation is on.